StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -471,6 +514,42 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
         ]
         return post(resource: "consumers/sessions/lookup", parameters: parameters)
     }
+
+    func consumerSessionStartVerification(
+        emailAddress: String,
+        otpType: String, // TODO(kgaidis): consider whether this should be an enum type SMS + EMAIL
+        customEmailType: String?, // TODO(kgaidis): consider whether this should be an enum type
```

Comment:
yes to enum?

Reply:
this API should be replaced by what bmelts is building, so I probably can even delete this comment (as this whole API code should be deleted), but I kept it there just to be extra 'careful'